src/main/scala/net/ripe/rpki/publicationserver/model/package.scala

```diff
@@ -0,0 +1,5 @@
+package net.ripe.rpki.publicationserver
+
+package object model {
```

Comment:
This package looks lonely with only ClientID. Maybe those in PublicationMessages can be moved here (all QueryPdu, Message sealed traits and enumerations).

Reply:
Yeah, makes sense.